src/app/shared/services/switchboard-toastr.service.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { TestBed } from '@angular/core/testing';
+
+import { SwitchboardToastr, SwitchboardToastrService } from './switchboard-toastr.service';
+import { ToastrModule, ToastrService } from 'ngx-toastr';
+import { BehaviorSubject } from 'rxjs';
+
+fdescribe('SwitchboardToastrService', () => {
+  let service: SwitchboardToastrService;
+  const toastrSpy = jasmine.createSpyObj('ToastrService',
+      [
+        'success',
+        'show',
+        'error',
+        'info',
+        'warning',
+      ]
+  );
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [{ provide: ToastrService, useValue: toastrSpy }],
+      imports: [ToastrModule.forRoot()]
+    });
+    service = TestBed.inject(SwitchboardToastrService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should be run reset', () => {
+    (service as any).massageList = new BehaviorSubject<SwitchboardToastr[]>([{
```

Comment:
I think we should test only public methods, not private. 
Private methods should be tested through public ones.